### PR TITLE
Update default app styles

### PR DESCRIPTION
**Pull request by kevinsawicki**
We are currently working on a redesign of http://electron.atom.io/ and this pull request matches the default app stylings to the upcoming look and feel of the new site.

| Before | After |
|----|---|
|screen shot 2016-04-28 at 9 23 00 am| screen shot 2016-04-28 at 9 21 55 am|

/cc @zeke @jlord @simurai

### CODE REVIEW

File: default_app/index.html [hunk 1]
```diff
@@ -53,18 +74,22 @@
...
 
     #holder {
-      border: 2px dashed #448691;
       margin: 0 auto;
-      height: 300px;
-      color: #45828E;
-      font-size: 40px;
-      line-height: 300px;
+      height: 275px;
+      border: 1px solid #e0e5e6;
```
Comment (blainesch) [inline default_app/index.html]: Should this be dashed to signify it's a special box? It currently has the same formatting as the command box.

Comment (kevinsawicki) [inline default_app/index.html]: I think the text "Drag your app here to run it" makes it pretty clear without a special border.

The border still changes color when something is dragged over it.

**Comment by zeke** [comment]
How does one see this screen in the first place? I forget.

**Comment by kevinsawicki** [comment]
> How does one see this screen in the first place? I forget.

Run `electron` with no args.

**Comment by zeke** [comment]
Tangent: I think it's kind of weird that this screen exists. Encouraging users to drag and drop things is not exemplary of how people familiar with electron actually use it. It would be cool to downplay this draggable business and make the "getting started" content more prominent.

**Comment by zeke** [comment]
Tangent aside, :shipit:

**Comment by simurai** [comment]
> It would be cool to downplay this draggable business

I guess you're saying that it looks more like a gimmick to a serious dev? :grin: Yeah, I know what you mean. But it might also makes Electron more "web designer friendly"? You can just drag&drop an `index.html` on it and have it shown in a new window. Not too useful and you can do it with a browser too, but it might still give some "wow, that's easy" effect?

We could just reduce it in height to make it a bit less prominent?

**Comment by YuriSolovyov** [comment]
Maybe it's better to just improve wording a bit to make it clearly state that d'n'd is optional and "real" approach is via `.asar` archives?

**Comment by kevinsawicki** [comment]
> Encouraging users to drag and drop things is not exemplary of how people familiar with electron actually use it

> I guess you're saying that it looks more like a gimmick to a serious dev?

> Maybe it's better to just improve wording a bit to make it clearly state that d'n'd is optional and "real" approach is via .asar archives?

Yeah, I agree taking a more content-focused pass at this screen would be worthwhile.

I'm going to merge this pull request since it was only a style change, not a content change, and we can revisit what content in the page would make the best first run experience in follow up pull requests.

**Pull request merged**